.travis.yml

```diff
@@ -123,9 +123,9 @@ jobs:
     env:
     - NODE_ENV=production
     - PORT=12345
-    - STREAMR_API_URL=https://ee-staging.streamr.com/api/v1
-    - STREAMR_WS_URL=wss://ee-staging.streamr.com/api/v1/ws
-    - STREAMR_URL=https://ee-staging.streamr.com
+    - STREAMR_API_URL=https://staging.streamr.com/api/v1
+    - STREAMR_WS_URL=wss://staging.streamr.com/api/v1/ws
+    - STREAMR_URL=https://staging.streamr.com
     - PLATFORM_ORIGIN_URL=https://marketplace-staging.streamr.com
```

Comment:
Should there be any references to https://marketplace-staging.streamr.com ? Shouldn't they all be https://staging.streamr.com